sagemaker-python-sdk/tensorboard_keras/tensorboard_keras.ipynb

```diff
@@ -134,45 +134,7 @@
    "cell_type": "markdown",
```

Comment:
<p>What is "the above command"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2956/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
changed sentence to :

Start Tensorboard from the System terminal and view the Tensorboard UI. To do this, follow steps 1-3 below: